content/faq/helm-vs-alternatives.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "What are the reasons for using Helm over some alternatives?"
+description: "Of the tools out there, Helm has probably a 1000x lead over the others."
+tags:
+- Helm
+- K8s
+- Kubernetes
+- YAML
+---
+
+## Question
+
+What are the reasons for using Helm over some alternatives (e.g. Draft, Gitkube, Ksonnet, Metaparticle, Scaffold, and Terraform)?
+
+
+## Answer
+
+Tiller potentially introduces well-known security concerns. There are plans for removing it, but no ETA yet when that will be released.
+
+The only constant in the K8s ecosystem is that it’s constantly changing and evolving. Helm is a very intrusive decision, and there are many alternative approaches. Of those, Ksonnet is most interesting, since it presents a pretty terse way of expressing Kubernetes resources and is the result of years of research into developing a concise language for JSON manipulation. Then again, coding in JSON isn't our fave either (Cloud Formation = evil).
```

Comment:
```suggestion
The only constant in the K8s ecosystem is that it’s constantly changing and evolving. Helm is one of the be options available today and there are many alternative approaches. Of those, Ksonnet is most interesting, since it presents a pretty terse way of expressing Kubernetes resources and is the result of years of research into developing a concise language for JSON manipulation. Then again, coding in JSON isn't our favorite pastime either; writing raw Cloud Formation is a horrible experience.
```